src/components/layout/footer.js

```diff
@@ -583,7 +583,11 @@ const Footer = () => {
                                 </Show.NonEU>
                                 <Show.Eu>
                                     <DisclaimerParagraph no_margin>
-                                        <Localize translate_text="The financial products offered on this website include options and contracts for difference (CFDs) which are considered complex derivatives and may not be suitable for everyone. CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage. 74% of retail investor accounts lose money when trading CFDs with Deriv Investments (Europe) Limited. You should consider whether you understand how CFDs work and whether you can afford to take the high risk of losing your money. The products mentioned here may be affected by changes in currency exchange rates. If you invest in these products, you may lose some or all of your investment and the value of your investment may fluctuate. You should never invest money that you cannot afford to lose and never trade with borrowed money." />
+                                        <Localize
+                                            translate_text="The financial products offered on this website include options and contracts for difference (CFDs) which are considered complex derivatives and may not be suitable for everyone.<br><br>
```

Comment:
<br> tag will not work anymore on firefox, anw, to add component/tag in the localize string:
```
<Localize translate_text="my <0>bold</0> text" components={[<strong key={0} />]}
```

Reply:
I was thinking there is no tag used here :3
converted the br tag to use components pattern.